[unstable] bubble.cpp fails to compile

```
In file included from /home/igor/Physics/TRIQS/libs.git/triqs/./gfs/gf/gf.hxx:28:
/home/igor/Physics/TRIQS/libs.git/triqs/./gfs/gf/./comma.hpp:59:15: error: chosen constructor is explicit in copy-initialization
      return {{std::get<Is>(std::move(tu._t))..., std::forward<X>(x)}};
              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/igor/Physics/TRIQS/libs.git/triqs/./gfs/gf/./comma.hpp:63:14: note: in instantiation of function template specialization 'triqs::gfs::__comma_impl<triqs::gfs::var_t &,
      triqs::gfs::mesh_point<gf_mesh<triqs::gfs::imfreq> >, triqs::gfs::mesh_point<gf_mesh<triqs::gfs::imfreq> > , 0, 1>' requested here
      return __comma_impl(std::move(t), std::forward<X>(x), std14::make_index_sequence<sizeof...(T)>());
             ^
/home/igor/Physics/TRIQS/libs.git/test/triqs/gfs/multivar/bubble.cpp:58:35: note: in instantiation of function template specialization 'triqs::gfs::operator,<triqs::gfs::var_t &,
      triqs::gfs::mesh_point<gf_mesh<triqs::gfs::imfreq> >, triqs::gfs::mesh_point<gf_mesh<triqs::gfs::imfreq> > >' requested here
   EXPECT_EQ((chi0q_from_r[inu, iw, _]).mesh().size(), 16);
                                  ^
/usr/lib/gcc/x86_64-pc-linux-gnu/5.4.0/include/g++-v5/tuple:479:19: note: constructor declared here
        constexpr tuple(_UElements&&... __elements)
```
My compiler is clang 4.0.1.
*N.B.* `explicit` specifier is removed from this constructor in C++17.

--- a/triqs/gfs/gf/comma.hpp
+++ b/triqs/gfs/gf/comma.hpp
@@ -56,7 +56,7 @@ namespace triqs {
     // _tuple absorbs anything
     template <typename X, typename... T, size_t... Is>
     FORCEINLINE _tuple<T..., std::decay_t<X>> __comma_impl(_tuple<T...> &&tu, X &&x, std14::index_sequence<Is...>) {
-      return {{std::get<Is>(std::move(tu._t))..., std::forward<X>(x)}};
+      return {std::make_tuple(std::get<Is>(std::move(tu._t))..., std::forward<X>(x))};
     }
 
     template <typename X, typename... T> FORCEINLINE _tuple<T..., std::decay_t<X>> operator,(_tuple<T...> &&t, X &&x) {

